incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -119,20 +127,32 @@ func RunErrorShouldContainMultiple(substrs []string, seconds float64, cmdln ...s
 	}, seconds).Should(Succeed())
 }
 
-func runShouldContainMultiple(substrs []string, cmdln ...string) ([]string, error) {
-		stdout, err := RunCommand(cmdln...)	
+func runShouldContainMultiple(substrs []string, cmdln ...string) (string, error) {
+		stdout, err := RunCommand(cmdln...)
 		GinkgoT().Log("Output: ", stdout)
 		return missAny(substrs, stdout), err
 }
 
-func missAny(substrs []string, teststring string) []string {
-	var missing []string 
+// If any of the substrs are missing from teststring, returns a string of the form:
+//   Missing: <string1>, <string2>, ...
+//   Got: teststring
+// Otherwise returns the empty string.
+func missAny(substrs []string, teststring string) string {
```

Comment:
Don't have an objection on it, but why do we prefer return a concatenated string than slice of strings?

Reply:
Good question. Usually, we should return structured data, but in this case, all the callers were doing exactly the same thing - displaying all the missing elements. I wanted to modify them all in the same way - not _just_ display what we _expected_, but also what we _got_. So it made the most sense to change the meaning of this function to return a single human-friendly _message_ to display, which now includes what we actually got.

If we were to find that in some cases we wanted the structured data (e.g. "ignore this missing string if that happens") then we'd have to break this function into two. But given how we were using it, returning a human-readable string made more sense.